tests/filesystem/realpath_test.php

```diff
@@ -1,90 +0,0 @@
-<?php
```

Comment:
I think you should keep both the old and new tests (even if they look duplicated)